targets/targets.json

```diff
@@ -3630,12 +3635,14 @@
             }
         },
         "overrides": {
-            "lse_available": 0
+            "lse_available": 0,
+            "boot-stack-size": "0x400",
```

Comment:
Why?
As explained in 
https://os.mbed.com/docs/mbed-os/v5.15/porting/porting-bootstrap.html
/* This value is normally defined by the tools to 0x1000 for bare metal and 0x400 for RTOS */

So why don't we keep default 0x1000 value ?
Thx